lib/whitehall/publishing_api.rb

```diff
@@ -55,6 +55,10 @@ def self.publish_gone(base_path)
       PublishingApiGoneWorker.perform_async(base_path)
     end
 
+    def self.discard_draft_async(edition)
+      PublishingApiDiscardDraftWorker.perform_async(edition.content_id, edition.translations.map(&:locale))
```

Comment:
This uses a different pattern to the other methods in this class, which map over `locales_for` and enqueue a job for each locale. Unless there is a good reason to do it differently, we should be consistent (and if there is a good reason, we should document it).